### DISCUSSION TITLE
How to get information about Left Mouse Button click event.

### DISCUSSION THREAD

**Discussion by deepspkd** [highfestiva/finplot#461] [Q&A] [answered]
I am trying to make a standalone windows app. Went through finplot's wiki - snippets, examples etc. and it helped me to draw chart, add indicators to it, draw lines and texts very easily with this wonderful library, even with my little python knowledge. 

I am using set_mouse_callback function for capturing left mouse click event and using it to draw lines where i wanted. But i realised that besides left click, middle and right click is also drawing the line. I am unable to filter out left mouse button click from other mouse button clicks. Is there any way i could get only Left Mouse Clicks. Please help.

**Comment by highfestiva** [comment, marked as answer]
Hadn't thought of that. Added support for it in 464fbbc. You use it like so:
```
fplt.set_mouse_callback(my_callback, ax=ax, when='lclick')
fplt.set_mouse_callback(my_callback, ax=ax, when='rclick')
fplt.set_mouse_callback(my_callback, ax=ax, when='mclick')
```

**Reply by deepspkd** [reply]
Thankyou
